bin/busted

```diff
@@ -1,3 +1,18 @@
-#!/usr/bin/env sh
+#!/usr/bin/env perl
```

Comment:
Is always good to:

```
use warnings;
use strict;
```

It protects you of a lot of stuff (https://perlmaven.com/always-use-warnings)

Reply:
Thx! Push'd.